src/Uno.UI/UI/Xaml/FrameworkElement.macOS.cs

```diff
@@ -155,7 +155,7 @@ protected virtual void OnAfterArrange()
 			return result.LogicalToPhysicalPixels();
 		}
 
-		public CGSize SizeThatFits(CGSize size)
+		public virtual CGSize SizeThatFits(CGSize size)
```

Comment:
This is most likely not needed as per the comment above.

Reply:
And fixed :-)